mapie/tests/test_regression.py

```diff
@@ -748,3 +748,20 @@ def test_predict_infinite_intervals() -> None:
     _, y_pis = mapie_reg.predict(X, alpha=0., allow_infinite_bounds=True)
     np.testing.assert_allclose(y_pis[:, 0, 0], -np.inf)
     np.testing.assert_allclose(y_pis[:, 1, 0], np.inf)
+
+
+@pytest.mark.parametrize("method", ["minmax", "naive", "plus"])
+@pytest.mark.parametrize("cv", ["split", "prefit"])
+def test_check_change_method_to_base(method: str, cv: str) -> None:
+    """Test of shift in power from one method to base method in fit"""
+
+    X_train, X_val, y_train, y_val = train_test_split(
+        X, y, test_size=0.5, random_state=random_state
+    )
+    estimator = LinearRegression().fit(X_train, y_train)
+    mapie_reg = MapieRegressor(
+        cv=cv, method=method, estimator=estimator
+    )
+    mapie_reg.fit(X_val, y_val)
+    assert mapie_reg.method == "base", \
+        f"Expected method base, but got {mapie_reg.method}"
```

Comment:
Here, all you have to do is check that the class has the right method. You don't have to handle any exceptions.
```suggestion
    assert mapie_reg.method == "base"
```